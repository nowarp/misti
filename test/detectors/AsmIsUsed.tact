asm(-> 1 0) extends mutates fun loadRefEx(self: Slice): Cell { LDREF }
