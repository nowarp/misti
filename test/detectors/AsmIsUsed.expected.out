[INFO] AsmIsUsed: asm function is used
test/detectors/AsmIsUsed.tact:1:1:
> 1 | asm(-> 1 0) extends mutates fun loadRefEx(self: Slice): Cell { LDREF }
      ^
  2 | 
Help: Using TVM assembly is a potentially dangerous operation that requires additional review
See: https://nowarp.io/tools/misti/docs/detectors/AsmIsUsed